src/Nethermind/Nethermind.Evm.Test/WithdrawalRequestsProcessorTests.cs

```diff
@@ -0,0 +1,99 @@
+// SPDX-FileCopyrightText: 2024 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System;
+using System.Linq;
+using FluentAssertions;
+using Nethermind.Consensus.Requests;
+using Nethermind.Core;
+using Nethermind.Core.ConsensusRequests;
+using Nethermind.Core.Extensions;
+using Nethermind.Core.Specs;
+using Nethermind.Core.Test.Builders;
+using Nethermind.Crypto;
+using Nethermind.Db;
+using Nethermind.Evm.Tracing;
+using Nethermind.Evm.TransactionProcessing;
+using Nethermind.Int256;
+using Nethermind.Logging;
+using Nethermind.Specs;
+using Nethermind.State;
+using Nethermind.Trie.Pruning;
+using NSubstitute;
+using NUnit.Framework;
+
+
+namespace Nethermind.Evm.Test;
```

Comment:
@rjnrohit why did you add it in Evm.Test, the class is in the Consensus.Test?

Reply:
for me it might be silly :) , but I wanted to re-use `TestBlockhashProvider` class from the  namespace `Nethermind.Evm.Test`, I tried putting this file in the `Consensus.Test` dir, but I couldn't import `TestBlockhashProvider` class